cloudapi/config.go

```diff
@@ -161,9 +164,10 @@ func NewConfig() Config {
 		MetricPushInterval:    types.NewNullDuration(1*time.Second, false),
 		MetricPushConcurrency: null.NewInt(1, false),
 
-		TracesEnabled:      null.NewBool(false, false),
-		TracesHost:         null.NewString("insights.k6.io:4443", false),
-		TracesPushInterval: types.NewNullDuration(1*time.Second, false),
+		TracesEnabled:         null.NewBool(false, false),
+		TracesHost:            null.NewString("insights.k6.io:4443", false),
+		TracesPushInterval:    types.NewNullDuration(10*time.Second, false),
+		TracesPushConcurrency: null.NewInt(6, false),
```

Comment:
It sounds like a magic number, do we have some math behind it?

Reply:
yeah, we could still default to `1` and see if we run into any issues

Reply:
Ok, I restored 1, thanks

Reply:
Does `1` mean no concurrency? Do you expect to tune this value during tests or ideally you would like to run with it as the default?

Reply:
yes, we would like to test first this value and then decide if we change ir.